governance/parameter-docs/param-dsr.md

```diff
@@ -0,0 +1,52 @@
+# $HumanReadableName
+
+```
+Alias: NA
+Parameter Name: Pot_dsr
+Containing Contract: Pot
+Scope: System
+Technical Docs:https://docs.makerdao.com/smart-contract-modules/rates-module/pot-detailed-documentation
+```
+
+## Description
+* What is this parameter?
```

Comment:
Remove these headers once you've filled in the section. Use the other completed parameter docs as a guide!